gen3/bin/awsrole.sh

```diff
@@ -188,6 +327,15 @@ gen3_awsrole() {
     'list' | 'ls')
       gen3_awsrole_list "$@"
       ;;
+    'create-sa')
+      gen3_awsrole_createsa "$@"
+      ;;
+    'create-assumerole')
+      gen3_awsrole_assumerole "$@"
+      ;;
+    'annotate-sa')
+      gen3_awsrole_annotatesa "$@"
+      ;;
     *)
```

Comment:
It would be good to add a test case to `gen3/test/lib/awsroleTest.sh` :
* delete test sa and role if they exist
* create a role
* create a service account
* annotate the sa
* delete everything